config/config.proto

```diff
@@ -48,9 +48,24 @@ message TargetGroup {
 	optional LabelPairs labels = 2;
 }
 
+// Information about the service discovery.
+message ServiceDiscovery {
+	// The name by which the service information can be found for
+	// the specified discovery service.
+	required string name = 1;
+	// The type of discovery to be used.
+	required string type = 2;
+	// Path to a configuration file that is used for setting up
+	// the discovery.
+	optional string config_file = 3;
+	// Discovery refresh period when using DNS-SD to discover targets. Must be a
+	// valid Prometheus duration string in the form "[0-9]+[smhdwy]".
+	optional string refresh_interval = 4 [default = "30s"];
+}
+
 // The configuration for a Prometheus job to scrape.
 //
-// The next field no. is 8.
+// The next field no. is 7.
```

Comment:
Actually, this wouldn't really matter for this protobuf which is never serialized/deserialized in binary form.